src/Features/VisualBasic/Portable/InlineHints/VisualBasicInlineParameterNameHintsService.vb

```diff
@@ -86,5 +86,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.InlineHints
 
             Return HintKind.Other
         End Function
+
+        Protected Overrides Function IsIndexer(node As SyntaxNode) As Boolean
+            Return False
```

Comment:
VB has the concept of default properties which can be thought of like indexers. I see this is C# only for right now. Are there plans to expand this to VB in the future?

Reply:
sure, I can add it right now. I wasn't sure if it followed indexers in c# 1:1, so I wasn't sure whether or not to add it.